.github/workflows/publish.yml

```diff
@@ -4,7 +4,7 @@ on: [push, pull_request]
 
```

Comment:
What's causing _some_ yaml keys to be quoted and some to not be? Is there a way to configure prettier to only put quotes when required?

Reply:
Actually, good question! I tinkered with Prettier a bit and it only quotes things that previously had quotes - we can drop the ones that only have 1 word w/out issue